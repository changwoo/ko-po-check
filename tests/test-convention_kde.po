# -*-text-*-
# test 9 - KDE
#
msgid ""
msgstr ""
"Project-Id-Version: test 9\n"
"POT-Creation-Date: 2009-04-13 01:30+0900\n"
"PO-Revision-Date: 2009-04-13 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# E: 번역자 써야 함
#: test.c:9
msgctxt "NAME OF TRANSLATORS"
msgid "Your names"
msgstr "귀하의 이름"

# E: 번역자 써야 함
#: test.c:9
msgctxt "NAME OF TRANSLATORS"
msgid "Your names"
msgstr "당신의 성함"

# E: 번역자 메일 써야 함
#: test.c:9
msgctxt "EMAIL OF TRANSLATORS"
msgid "Your emails"
msgstr "이메일"


# E: 번역자 메일 써야 함
#: test.c:9
msgctxt "EMAIL OF TRANSLATORS"
msgid "Your emails"
msgstr "편지 주소"
