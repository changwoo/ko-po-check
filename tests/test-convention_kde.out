-:19: KDE 번역 규칙에 따라 번역자 이름을 써야 합니다
-:25: KDE 번역 규칙에 따라 번역자 이름을 써야 합니다
-:31: KDE 번역 규칙에 따라 번역자 메일 주소를 써야 합니다
-:38: KDE 번역 규칙에 따라 번역자 메일 주소를 써야 합니다
